components/angular-tomitribe-tags/src/tags.configurer.ts

```diff
@@ -1,3 +1,12 @@
 export class TagsConfigurer {
-  tagsEndpoint = 'api/label/available';
+    tagsEndpoint = 'api/label/available';
+
+    pattern: string = "^[A-Za-z0-9 ._-]*$";
```

Comment:
then we can use it as `< ... tribe-tagging-validator ... >` (meaning default) or `< ... tribe-tagging-validator='numberPositive' ... >`